lib/lev/routine.rb

```diff
@@ -510,5 +490,38 @@ def new_term_mapper(options)
       nil
     end
 
+    module ResqueStatusClassMethods
+      def self.extended(base)
+        base.send(:include, Resque::Plugins::Status)
+        base.class_eval do
+          define_method(:initialize) do |uuid = nil, options = {}|
+            super(uuid, options)
+          end
+        end
+      end
+
+      def perform_later(*args, &block)
+        create(*args, &block)
+      end
+    end
+
+    module ResqueClassMethods
+      def perform_later(*args, &block)
+        Resque.enqueue(self, *args, &block)
+      end
+    end
+
+    private
+    def self.enable_background_jobs(base)
+      base.instance_eval do
+        @queue = :default
```

Comment:
When changing this to reflect value of `background_queue` as discussed above, can you write a spec that sets a unique name for a queue in a routine and then tests that [Resque#queue_from_class](http://www.rubydoc.info/github/resque/resque/Resque:queue_from_class) actually returns that queue name?